### PR TITLE
Add ZTE F6005v3 ONT

### PR DESCRIPTION
**Pull request by stich86**
Add new ZTE F6005v3 ONT info

### CODE REVIEW

**Review by simonebortolin — APPROVED** [review]

File: _ont/ont-zte-f6005v3.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+| Vendor/Brand | ZTE                                                               |
+| Model        | F6005v3                                                           |
+| ODM          | ✅                                                                |
+| CPU          | ZTE ZX279133@A53                                                  |
+| CPU Clock    | 2x1200MHz                                                         |
+| Chipset      | ZTE ZX279133                                                      |
+| Flash        | 128 MB (SPI NAND FM25S01A)                                        |
+| RAM          | 128 MB                                                            |
+| System       | Customized Linux by ZTE                                           |
+| 2.5GBaseT    | No                                                                |
```
Comment (davidebeatrici) [inline _ont/ont-zte-f6005v3.md:21]:
```suggestion
| 2.5GBaseT    | Yes                                                               |
```

File: _ont/ont-zte-f6005v3.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+```sh
+syn_version
+```
+
+# Use
+{% include alert.html content="Commands have been tested on V3 HW rev. on OpenFiber firmwares" alert="Note"  icon="svg-info" color="blue" %}
+
+## Enable Telnet
+{% include alert.html content="This is an external script ([ZTE Telnet enabler](https://github.com/douniwan5788/zte_modem_tools)), use at your own risk! Credentials don't survive at reboot!" alert="Note"  icon="svg-info" color="blue" %}
```
Comment (davidebeatrici) [inline _ont/ont-zte-f6005v3.md:78]: Do the GPON parameters (e.g. PLOAM) remain, though?

File: _ont/ont-zte-f6005v3.md (hunk 1 above)
Comment (stich86) [reply _ont/ont-zte-f6005v3.md:21]: My fault, I've copied F601 template and missed this one 😅

I've some other fresh news, so I'll fix on the next pull. Thanks 😊

File: _ont/ont-zte-f6005v3.md (hunk 2 above)
Comment (stich86) [reply _ont/ont-zte-f6005v3.md:78]: > Do the GPON parameters (e.g. PLOAM) remain, though?

Yes setmac commands are permanent 😊

Comment (davidebeatrici) [reply _ont/ont-zte-f6005v3.md:78]: Excellent!

File: _ont/ont-zte-f6005v3.md (hunk 1 above)
Comment (davidebeatrici) [reply _ont/ont-zte-f6005v3.md:21]: Thank you for your research!

**Pull request merged**